lms/static/js/student_account/views/RegisterView.js

```diff
@@ -102,6 +154,94 @@
                     return this;
                 },
 
+                postRender: function() {
+                    var inputs = [
+                            $('#register-name'),
+                            $('#register-email'),
+                            $('#register-username'),
+                            $('#register-password'),
+                            $('#register-country')
+                        ],
+                        inputTipSelectors = ['tip error', 'tip tip-input'],
+                        focusinStyles = {
+                            position: 'relative',
+                            'padding-top': '0px',
+                            'padding-left': '0px',
+                            opacity: '1.0'
+                        },
+                        focusoutStyles = {
+                            position: 'absolute',
+                            'padding-top': '5px',
+                            'padding-left': '9px',
+                            opacity: '0.75',
+                            'z-index': '1'
+                        },
```

Comment:
Generally styling with CSS is preferred (so JS is just adding/removing classes).